<!doctype html><html lang="en" class="height-full" data-color-mode="auto" data-light-theme="light" data-dark-theme="dark"><head><meta charset="utf-8"><meta name="viewport" content="width=device-width,initial-scale=1"><link rel="stylesheet" href="/assets/dist/styles.css"><link rel="apple-touch-icon" sizes="180x180" href="/apple-touch-icon.png?v=jw3d9vxKo4"><link rel="icon" type="image/png" sizes="32x32" href="/favicon-32x32.png"><link rel="icon" type="image/png" sizes="16x16" href="/favicon-16x16.png"><link rel="manifest" href="/site.webmanifest?v=jw3d9vxKo4"><link rel="mask-icon" href="/safari-pinned-tab.svg?v=jw3d9vxKo4" color="#5bbad5"><link rel="shortcut icon" href="/i3s.ico"><meta name="theme-color" content="#66cce0"></head><body><div class=""><div class="container-lg d-flex flex-items-center flex-justify-between px-3"><div class="py-3 d-inline-flex flex-items-center"><div class="col-1"><a href="/"><img src="/assets/img/i3s.jpg" class="width-fit" alt="i3S logo"></a></div><div class="text-inherit text-left"><h1 class="h4 text-uppercase">Bioinformatic Workflows</h1></div></div><details class="dropdown details-reset details-overlay d-inline-block focus-within"><summary class="h1" aria-haspopup="true">â‰¡</summary><div class="dropdown-menu dropdown-menu-w"><li><details class="dropdown details-reset details-overlay"><summary class="" aria-haspopup="true"><p class="dropdown-item" style="margin-bottom:0">Genomics</p></summary><div class="dropdown-menu dropdown-menu-w px-3"><li><a class="dropdown-item" href="/analysis/genomics/genome_assembly/">Genome assembly</a></li><li><a class="dropdown-item" href="/analysis/genomics/variant_detection/">Variant analysis</a></li><li><a class="dropdown-item" href="/analysis/genomics/metagenomic_analysis/">Metagenomics</a></li><li><a class="dropdown-item" href="/analysis/genomics/genetic_analysis/">Genetic analysis</a></li></div></details></li><li><details class="dropdown details-reset details-overlay"><summary class="" aria-haspopup="true"><p class="dropdown-item" style="margin-bottom:0">Transcriptomics</p></summary><div class="dropdown-menu dropdown-menu-w px-3"><li><a class="dropdown-item" href="/analysis/transcriptomics/exploratory_analysis/">Exploratory gene expression analysis</a></li><li><a class="dropdown-item" href="/analysis/transcriptomics/rna_seq_differential_expression_analysis/">RNA-seq analysis</a></li><li><a class="dropdown-item" href="/analysis/transcriptomics/pathway_analysis/">Pathway analysis</a></li><li><a class="dropdown-item" href="/analysis/transcriptomics/transcriptome_assembly/">Transcriptome assembly</a></li><li><a class="dropdown-item" href="/analysis/transcriptomics/single_cell_expression_analysis/">Single-cell expression analysis</a></li><li><a class="dropdown-item" href="/analysis/transcriptomics/microRNA_data_analysis/">MicroRNA data analysis</a></li><li><a class="dropdown-item" href="/analysis/transcriptomics/alternative_splicing_analysis/">Alternative splicing analysis</a></li><li><a class="dropdown-item" href="/analysis/transcriptomics/fusion_gene_detection/">Fusion gene detection</a></li></div></details></li></div></details></div></div><div class="container-lg px-3"><div class="d-md-flex gutter-md-spacious"><div class="col-md-auto"><div class="markdown-body py-3"><h2>RNAseq STAR (Default settings)</h2><p>This pipeline performs the alignment of fastq files, from RNA-seq reads, using the STAR aligner. This software is currently the one with the higher accuracies, even though it requires a large amount of RAM (50Gb for the human genome)</p><h3>What you should provide</h3><ol><li>Reads in fastq format (ideally compressed in .gz format)</li><li>The organism from which they come from.</li></ol><h3>What you will receive</h3><ol><li>File with the aligned reads in BAM format.</li><li>File with gene quantification for the sample.</li><li>File with alignment statistics.</li><li>Quality control plots representing the various factors affecting sample overall quality</li></ol><p><strong>WARNING:</strong> These files are required for followup analysis, such as <a href="../visual_representations/principal_component_analysis/">Principal component analysis</a>, <a href="../visual_representations/heatmap/">Heatmap analysis</a>, <a href="../exploratory_analysis/differentia_expression_analysis/">Differential expression analysis</a> and <a href="../pathway_analysis/">Pathway analysis</a> Visual exploration of the BAM genomic data can be assessed in <a href="https://software.broadinstitute.org/software/igv/">IGV</a>.</p><h3>Software versions</h3><ul><li>STAR v2.7.10a</li><li>BioBamBam2 v2.0.180</li><li>RSeQC v4.0.0</li><li>dupRadar v1.26.1</li></ul><h3>Relevant software values (assume default if unspecified)</h3><ul><li>alignSJoverhangMin set to <strong>8</strong></li><li>alignSJDBoverhangMin set to <strong>1</strong></li></ul><h3>Pipeline <a href="https://raw.githubusercontent.com/i3S-BioinformaticsService/i3s-cwl-ngs-workflows/main/RNA-Seq/rnaseq.star.default.json">ðŸ”—</a></h3><div id="vue" style="height: 500px;"><cwl cwl-url="https://raw.githubusercontent.com/i3S-BioinformaticsService/i3s-cwl-ngs-workflows/main/RNA-Seq/rnaseq.star.default.json"></cwl></div><script src="/assets/js/vue.min.js"></script><script src="/assets/js/cwl_svg.js"></script><script src="/assets/js/tua-bsl.umd.min.js"></script><script>new Vue({
           el: '#vue',
           components: {
             cwl: vueCwl.default
          }
 });
  var $ = document.querySelector.bind(document)
      $('#vue').onmouseover = function () {bodyScrollLock.lock('#vue')}
      $('#vue').onmouseout = function() { bodyScrollLock.unlock('#vue')}</script><h3>Methods (Please adapt)</h3></div></div></div></div></body></html>