---
layout: default
title: De novo sequencing

eleventyNavigation:
  key: Transcriptome assembly
  parent: Transcriptomics
  order: 4
---

<script> function mouseOver(x) {document.getElementById("legend").innerHTML = x.title;}</script>

<div class="color-bg-tertiary-light">
  <div class="container-md" style="flex-basis:66.66%" >
    <img
      alt=""
      src="/assets/img/rna_seq_transcriptome_assembly.png"
      onmouseover="mouseOver(this)"
      class="block width-fit"
      title="Exploratory analysis"
          />
  </div>

  <div class="container-lg d-flex flex-column justify-content-evenly py-3 px-3">
    <style> div {text-align: justify; text-justify: inter-word; } </style> 
    
    <h1 class="h2">Transcriptome assembly</h1>
    <p class="f3">
      Transcriptome assembly aims to reconstructute the complete set of transcripts that are expressed in a determined sample. Assembly methods can be classified in two categories: de novo assembly and genome-guided assembly. The first uses solely the RNAseq reads to generate the assembly while the latter can make use of a reference genome or transcriptome to aid the assembly process. Reference-guided assembly is particularly useful for the cases where a reference genome of a closely related species exist. In these cases, the alignment to paralogs or other shared sequences can help partition the reads prior to assembly which can help increase the assembly quality and completeness. For highly fragmented draft genomes, a de novo transcriptome assembly strategy is recommended.
    </p>

    <h1 class="h2 pt-4">Workflow</h1>
    <div class="container-lg page-section pt-2">
      <div class="container-lg d-flex flex-column flex-sm-row flex-sm-justify-between flex-sm-items-center py-3 px-3">
	<div class="d-flex flex-self-center" style="flex-basis:10%">
	  <h3 class="h3 text-center mb-auto">Step 1</h3>
	</div>
	
	<div class="d-flex flex-column" style="flex-basis:90%" >
	  <h2 class="h3">Pre-assembly quality control and filtering</h2>
	  <p class="f3">
	    The first step involves a quality assessement of the raw read data by the evaluation of base quality distribution, kmer frequencies and adapter contamination parameters. These parameters provide an indication of underlying quality and should be addressed before the assembly step to aid increase the overall quality of the assembled transcriptome. After the above cleaning process, ribossomal RNA (rRNA) reads, which can be represented in sizable quantities, should be removed.
	  </p>
	</div>
      </div>

      <div class="container-lg d-flex flex-column flex-sm-row flex-sm-justify-between flex-sm-items-center py-3 px-3">
	<div class="d-flex flex-self-center" style="flex-basis:10%">
	  <h3 class="h3 text-center mb-auto">Step 2</h3>
	</div>
	
	<div class="d-flex flex-column" style="flex-basis:90%" >
	  <h2 class="h3">Transcriptome read assembly</h2>
	  <p class="f3">
	    For the transcriptome assembly step, literature recommens the comparison of at least two different assemblers tools and multiple k-mer lengths, the latter a major trade-off to take into consideration in the assembly process. For example, sorter k-mer lengths contribute to the recovery of lowly expressed transcripts, but increase the number of incorrect/non-existent contigs. Longer k-mer length reduce the total number of contigs assembled, but also suppress the recovery of lowly expressed transcripts.
	  </p>
	</div>
      </div>

      <div class="container-lg d-flex flex-column flex-sm-row flex-sm-justify-between flex-sm-items-center py-3 px-3">
	<div class="d-flex flex-self-center" style="flex-basis:10%">
	  <h3 class="h3 text-center mb-auto">Step 3</h3>
	</div>
	
	<div class="d-flex flex-column" style="flex-basis:90%" >
	  <h2 class="h3">Assembly quality control and completeness</h2>
	  <p class="f3">
	    The quality of an assembly will be assessed from several perspectives: 1) sequence length (N50/ExN50) and fragmentation; 2) the fraction of all reads that map back to the assembly (recommended &#62; 80% total reads) and 3) assembly composition, which consists in testing for the presence of certain genes orthologs that are universal, persistently expressed and occur almost exclusively as single copies in the genome.
	  </p>  
	</div>
      </div>
      <div class="container-lg d-flex flex-column flex-sm-row flex-sm-justify-between flex-sm-items-center py-3 px-3">
	<div class="d-flex flex-self-center" style="flex-basis:10%">
	  <h3 class="h3 text-center mb-auto">Step 4</h3>
	</div>
	
	<div class="d-flex flex-column" style="flex-basis:90%" >
	  <h2 class="h3">Annotation</h2>
	  <p class="f3">
	    Annotation of the assembly genome can be achieved by a combination of different approaches. The most common involve 1) searching for homologs based on sequence similaraty, 2) searching for functional sequence features (e.g. protein domains, motifs, transmembrane helices) identification and 3) assigning gene ontology (GO) terms to the sequences.
	  </p>  
	</div>
      </div>
    </div>
    
    <div class="container-lg d-flex flex-column justify-content-evenly pt-4">
      <h1 class="h2">Available pipelines</h1>
      <div class="container-lg page-section py-2">
	<div class="d-md-flex flex-wrap gutter flex-auto">
	  {% assign apps = collections.genomics %}
	  {% for app in apps %}{% if 'rna_read_assembly' == app.data.label %}
	  {% include "genomic" app: app.data %}{% endif %}
	  {% endfor %}
	</div>
      </div>
    </div>


    <h1 class="h2 pt-4">Follow-up analysis</h1>
    <div class="container-lg page-section py-2">
      <div class="d-md-flex flex-wrap gutter flex-auto">
	<div class="d-flex flex-column" style="flex-basis:33.33%">
	  <div class="d-flex flex-self-center anim-hover-grow" >
	    <a href="/analysis/transcriptomics/rna_seq_differential_expression_analysis/">
	      <img
		alt=""
		src="/assets/img/rna_seq_exploratory_analysis.png"
		onmouseover="mouseOver(this)"
		class="block width-fit"
		title="Small variant analysis"
		/>
	    </a>
	  </div>
	  <div class="page-section d-flex flex-column mt-auto">
	    <h3 class="h3 text-center mb-auto">RNA-seq analysis</h3>    
	  </div>
	</div>
      </div>
    </div>

    <h1 class="h2 pt-4">Visual representations</h1>
    <div class="container-lg page-section py-2">
      <div class="d-md-flex flex-wrap gutter flex-auto">
	<div class="d-flex flex-column" style="flex-basis:33.33%">
	  <div class="d-flex flex-self-center anim-hover-grow" >
	    <a href="/analysis/genomics/de_novo_sequencing2/">
	      <img
		alt=""
		src="/assets/img/circos.png"
		onmouseover="mouseOver(this)"
		class="block width-fit"
		title="Circos plot"
		/>
	    </a>
	  </div>
	  <div class="page-section d-flex flex-column mt-auto">
	    <h3 class="h3 text-center mb-auto">Circos plot</h3>    
	  </div>
	</div>
      </div>
    </div>
  </div>
</div>
