---
layout: default
title: Copy-number variant analysis
---

<script> function mouseOver(x) {document.getElementById("legend").innerHTML = x.title;}</script>

<div class="color-bg-tertiary-light">
  <div class="container-lg text-center" >
    <img
      alt=""
      src="/assets/img/dnaseq/copy_number_analysis.large.png"
      onmouseover="mouseOver(this)"
      class="block width-fit"
      title="Copy number analysis"
          />
  </div>

  <div class="container-lg d-flex flex-column justify-content-evenly py-3 px-3">
    <style> div {text-align: justify; text-justify: inter-word; } </style> 
    
    <h1 class="h2">Copy-number variant analysis</h1>
    <p class="f3">
      Copy number variants (CNVs) are regions of the genome containing additional DNA copies (duplications or amplifications) or losses of genetic material (deletions). These variants have been identified across all domains of life, from bacteria and archaea to plants and animals and are and important source of genetic diversity. Although CNVs are a subtype of structural genome variants, their definition and criteria are still evolving. Currently, the size of CNVs is defined from 50bp to several MB.
    </p>

    <h1 class="h2 pt-4">Workflow</h1>
    <div class="container-lg page-section pt-2">
      <div class="container-lg d-flex flex-column flex-sm-row flex-sm-justify-between flex-sm-items-center py-3 px-3">
	<div class="d-flex flex-self-center" style="flex-basis:10%">
	  <h3 class="h3 text-center mb-auto">Step 1</h3>
	</div>
	
	<div class="d-flex flex-column" style="flex-basis:90%" >
	  <h2 class="h3">Quality control</h2>
	  <p class="f3">
	Raw sequence data obtained from a sequencing service provider, in general, is not immediately ready for variant discovery. A first quality control (QC) step is recommended for the assessement of contaminations, bias and errors in raw sequences. By addressing these issues we can reduce the impact of these errors in downstream analysis.  
      </p>
	</div>
      </div>
      
      <div class="container-lg d-flex flex-column flex-sm-row flex-sm-justify-between flex-sm-items-center py-3 px-3">
	<div class="d-flex flex-self-center" style="flex-basis:10%">
	  <h3 class="h3 text-center mb-auto">Step 2</h3>
	</div>
	
	<div class="d-flex flex-column" style="flex-basis:90%" >
	  <h2 class="h3">Sequence alignment</h2>
	  <p class="f3">
	    In this step, reads are aligned to a reference genome to determine its precise location.
	  </p>
	</div>
      </div>
      
      <div class="container-lg d-flex flex-column flex-sm-row flex-sm-justify-between flex-sm-items-center py-3 px-3">
	<div class="d-flex flex-self-center" style="flex-basis:10%">
	  <h3 class="h3 text-center mb-auto">Step 3</h3>
	</div>
	
	<div class="d-flex flex-column" style="flex-basis:90%" >
	  <h2 class="h3">Post-alignment processing</h2>
	  <p class="f3">
	    After the alignment step, technical biases from the underlying sequencing platform and or sample preparation steps can still linger which affect variant calling. Two of the most common sources of biases are duplicates (PCR duplicates and optical duplicates) and uneven base quality scores between sequencing cycles. By identifying duplicates and recalibrating base quality scores we can reduce the impact of these sources of bias from the analysis.
      </p>
	</div>
      </div>

      <div class="container-lg d-flex flex-column flex-sm-row flex-sm-justify-between flex-sm-items-center py-3 px-3">
	<div class="d-flex flex-self-center" style="flex-basis:10%">
	  <h3 class="h3 text-center mb-auto">Step 4</h3>
	</div>
	
	<div class="d-flex flex-column" style="flex-basis:90%" >
	  <h2 class="h3">Copy-number discovery</h2>
	  <p class="f3">
	    At this stage, identification of copy-numbers can be assertained using the post-alignment processed file. CNV detection methods can be categorized into five different strategies, including: (1) paired-end mapping (PEM), (2) split read (SR), (3) read depth (RD), (4) de novo assembly of a genome (AS). Each of these approaches has strenghts and limitations depending on the size of CNV and genome coverage. For this reason, a combinatorial approach of the different algorithms is typically recommended. 
	  </p>  
	</div>
      </div>

      <div class="container-lg d-flex flex-column flex-sm-row flex-sm-justify-between flex-sm-items-center py-3 px-3">
	<div class="d-flex flex-self-center" style="flex-basis:10%">
	  <h3 class="h3 text-center mb-auto">Step 5</h3>
	</div>
	
	<div class="d-flex flex-column" style="flex-basis:90%" >
	  <h2 class="h3">Filtration of variants</h2>
	  <p class="f3">
	    After CNV discovery filtration is recommended to reduce the numer of false positives resulting from experimental biases. Two such examples of biases are the GC content and the presence of repetitive DNA elements. The first can lead to certain genomic regions being over or under-sampled. The latter can result in large differences between the number of unambiguously mapped reads to a region the number of reads sequenced from that region.
	  </p>  
	</div>
      </div>
      
      <div class="container-lg d-flex flex-column flex-sm-row flex-sm-justify-between flex-sm-items-center py-3 px-3">
	<div class="d-flex flex-self-center" style="flex-basis:10%">
	  <h3 class="h3 text-center mb-auto">Step 6</h3>
	</div>
	
	<div class="d-flex flex-column" style="flex-basis:90%" >
	  <h2 class="h3">Variant annotation</h2>
	  <p class="f3">
	     At this stage, filtered CNVs, typically defined as: delection, loss, normal, gain or amplification will be annotated regarding their gene(s)/transcript(s).
	  </p>  
	</div>
      </div>
    </div>
    
    <div class="container-lg d-flex flex-column justify-content-evenly px-3 pt-4">
      <h1 class="h2">Available pipelines</h1>
      <div class="container-lg page-section py-2">
	<div class="d-md-flex flex-wrap gutter flex-auto">
	  {% assign apps = collections.genomics %}
	  {% for app in apps %}{% if 'copy_number' == app.data.label %}
	  {% include "genomic" app: app.data %}{% endif %}
	  {% endfor %}
	</div>
      </div>
    </div>
  </div>
</div>
