---
layout: default
title: 16s metagenomics
---

<script> function mouseOver(x) {document.getElementById("legend").innerHTML = x.title;}</script>

<div class="color-bg-tertiary-light">
  <div class="container-lg text-center" >
    <img
      alt=""
      src="/assets/img/dnaseq/metagenomics_analysis.large.png"
      onmouseover="mouseOver(this)"
      class="block width-fit"
      title="Metagenomics by marker gene sequencing"
      />
  </div>
  
  <div class="container-lg d-flex flex-column justify-content-evenly py-3 px-3">
    <style> div {text-align: justify; text-justify: inter-word;} </style> 
    
    <h1 class="h2">Metagenome through marker gene sequencing</h1>
    <p class="f3">
      Metataxonomics provide a cost-effective means to identify a wide range of organisms since they rely on rRNA gene sequences for taxonomic profiling. These rRNA markers include 16S rRNA for bacteria, 18S rRNA for eukaryotes, and the internal transcribed spacer (ITS) regions of the fungal ribosome for fungi. These markers work well for phylogenetic profiling because they are ubiquitously present in the population, they have hypervariable regions that differentiate species and they are flanked by conserved regions that can be targeted by "universal" primers. In the analysis of these sequences, two major approaches for representative sequence selections can be employed, clustering to operational taxonomic units (OTUs) and denoising to amplicon sequence variants (ASVs). ASVs have been proposed to replace the limitations in OTU clustering.
    </p>
    
    <h1 class="h2 pt-4">Workflow</h1>
    <div class="container-lg page-section pt-2">
      <div class="container-lg d-flex flex-column flex-sm-row flex-sm-justify-between flex-sm-items-center py-3 px-3">
	<div class="d-flex flex-self-center" style="flex-basis:10%">
	  <h3 class="h3 text-center mb-auto">Step 1</h3>
	</div>
	
	<div class="d-flex flex-column" style="flex-basis:90%" >
	  <h2 class="h3">Merging, removal of barcodes & primers, and quality control</h2>
	  <p class="f3">
	    In this first stage, raw reads will be demultiplexed (i.e assigning barcoded reads to the respective sample) and a quality filter will be imposed. The quality filtering parameters impose limits on the: (i) minimum number of consecutive high-quality base calls; (ii) maximum number of consecutive low-quality base calls; (iii) maximum number of ambigous (N) characters allowed in a sequence and (iv) the minimum allowed base quality score (Phred quality score). At last paired-end reads, when applicable) are merged to obtain amplicon sequences, and barcodes and primers are removed.      
	  </p>
	</div>
      </div>
      
      <div class="container-lg d-flex flex-column flex-sm-row flex-sm-justify-between flex-sm-items-center py-3 px-3">
	<div class="d-flex flex-self-center" style="flex-basis:10%">
	  <h3 class="h3 text-center mb-auto">Step 2</h3>
	</div>
	
	<div class="d-flex flex-column" style="flex-basis:90%" >
	  <h2 class="h3">Clustering/Denoising</h2>
	  <p class="f3">
	    Picking representative sequences as proxies for species is a key step of these types of amplicon analysis. Two major representative approaches are available, OTU clustering and ASV denoising. 
	  </p>
	</div>
      </div>
      
      <div class="container-lg d-flex flex-column flex-sm-row flex-sm-justify-between flex-sm-items-center py-3 px-3">
	<div class="d-flex flex-self-center" style="flex-basis:10%">
	  <h3 class="h3 text-center mb-auto">Step 3</h3>
	</div>
	
	<div class="d-flex flex-column" style="flex-basis:90%" >
	  <h2 class="h3">Quantification</h2>
	  <p class="f3">
	    A feature table is obtained quatifying the frequency of the feature sequences in each sample. To this feature table, a taxonomical assigment will be performed (kingdom, phylum, class, order, family, genus and species).
	  </p>
	</div>
      </div>

      <div class="container-lg d-flex flex-column flex-sm-row flex-sm-justify-between flex-sm-items-center py-3 px-3">
	<div class="d-flex flex-self-center" style="flex-basis:10%">
	  <h3 class="h3 text-center mb-auto">Step 4</h3>
	</div>
	
	<div class="d-flex flex-column" style="flex-basis:90%" >
	  <h2 class="h3">Diversity analysis</h2>
	  <p class="f3">
	    At this stage, alpha and beta diversity will be evalutated to assess microbiota diversity. A phylogenetic tree will be generated based on the detect species. Co-occurence, correlation and discrimination of phylotypes by condition or group will also be carried, if applicable. 
	  </p>
	</div>
      </div>
      
      <div class="container-lg d-flex flex-column flex-sm-row flex-sm-justify-between flex-sm-items-center py-3 px-3">
	<div class="d-flex flex-self-center" style="flex-basis:10%">
	  <h3 class="h3 text-center mb-auto">Step 5</h3>
	</div>
	
	<div class="d-flex flex-column" style="flex-basis:90%" >
	  <h2 class="h3">Functional prediction</h2>
	  <p class="f3">
	   In this last step, metabolic predictions and functional annotations for the collections of phylotypes will be evaluated.
	  </p>  
	</div>
      </div>
    </div>
    
    <div class="container-lg d-flex flex-column justify-content-evenly px-3 pt-4">
      <h1 class="h2">Available pipelines</h1>
      <div class="container-lg page-section py-2">
	<div class="d-md-flex flex-wrap gutter flex-auto">
	  {% assign apps = collections.genomics %}
	  {% for app in apps %}{% if 'metagenomics_16s' == app.data.label %}
	  {% include "genomic" app: app.data %}{% endif %}
	  {% endfor %}
	</div>
      </div>
    </div>
  </div>
</div>
