<!doctype html><html lang="en" class="height-full" data-color-mode="auto" data-light-theme="light" data-dark-theme="dark"><head><meta charset="utf-8"><meta name="viewport" content="width=device-width,initial-scale=1"><link rel="stylesheet" href="/assets/dist/styles.css"><link rel="apple-touch-icon" sizes="180x180" href="/apple-touch-icon.png?v=jw3d9vxKo4"><link rel="icon" type="image/png" sizes="32x32" href="/favicon-32x32.png"><link rel="icon" type="image/png" sizes="16x16" href="/favicon-16x16.png"><link rel="manifest" href="/site.webmanifest?v=jw3d9vxKo4"><link rel="mask-icon" href="/safari-pinned-tab.svg?v=jw3d9vxKo4" color="#5bbad5"><link rel="shortcut icon" href="/i3s.ico"><meta name="theme-color" content="#66cce0"></head><body><div class=""><div class="container-lg d-flex flex-items-center flex-justify-between px-3"><div class="py-3 d-inline-flex flex-items-center"><div class="col-1"><a href="/"><img src="/assets/img/i3s.jpg" class="width-fit" alt="i3S logo"></a></div><div class="text-inherit text-left"><h1 class="h4 text-uppercase">Bioinformatic Workflows</h1></div></div><details class="dropdown details-reset details-overlay d-inline-block focus-within"><summary class="h1" aria-haspopup="true">â‰¡</summary><div class="dropdown-menu dropdown-menu-w"><li><details class="dropdown details-reset details-overlay"><summary class="" aria-haspopup="true"><p class="dropdown-item" style="margin-bottom:0">Genomics</p></summary><div class="dropdown-menu dropdown-menu-w px-3"><li><a class="dropdown-item" href="/analysis/genomics/genome_assembly/">Genome assembly</a></li><li><a class="dropdown-item" href="/analysis/genomics/variant_detection/">Variant analysis</a></li><li><a class="dropdown-item" href="/analysis/genomics/metagenomic_analysis/">Metagenomics</a></li><li><a class="dropdown-item" href="/analysis/genomics/genetic_analysis/">Genetic analysis</a></li></div></details></li><li><details class="dropdown details-reset details-overlay"><summary class="" aria-haspopup="true"><p class="dropdown-item" style="margin-bottom:0">Transcriptomics</p></summary><div class="dropdown-menu dropdown-menu-w px-3"><li><a class="dropdown-item" href="/analysis/transcriptomics/exploratory_analysis/">Exploratory gene expression analysis</a></li><li><a class="dropdown-item" href="/analysis/transcriptomics/rna_seq_differential_expression_analysis/">RNA-seq analysis</a></li><li><a class="dropdown-item" href="/analysis/transcriptomics/pathway_analysis/">Pathway analysis</a></li><li><a class="dropdown-item" href="/analysis/transcriptomics/transcriptome_assembly/">Transcriptome assembly</a></li><li><a class="dropdown-item" href="/analysis/transcriptomics/single_cell_expression_analysis/">Single-cell expression analysis</a></li><li><a class="dropdown-item" href="/analysis/transcriptomics/microRNA_data_analysis/">MicroRNA data analysis</a></li><li class=""><a class="dropdown-item" href="/analysis/transcriptomics/alternative_splicing_analysis/">Alternative splicing analysis</a></li><li><a class="dropdown-item" href="/analysis/transcriptomics/fusion_gene_detection/">Fusion gene detection</a></li></div></details></li></div></details></div></div><script>function mouseOver(x) {document.getElementById("legend").innerHTML = x.title;}</script><div class="color-bg-tertiary-light"><div class="container-lg text-center"><img alt="" src="/assets/img/rnaseq/rna_seq_alternative_splicing_analysis.png" class="block" title="Alternative splicing" style="max-width: 300px;"></div><div class="container-lg d-flex flex-column justify-content-evenly py-3 px-3"><style>div {text-align: justify; text-justify: inter-word; }</style><h1 class="h2">Alternative splicing analysis</h1><p class="f3">Alternative splicing events are pervasive in eukaryotes generating distint mRNAs by a differentially selection mechanisms within the pre-messager RNA. Alternative splicing mechanisms can include cassete exon, alternative 5' splice site, alternative 3' splice site, mutually exclusive exons, exon scrambling, intron retention, alternative promoter and alternative polyadenylation events. Methodologies for detection and analysis of these alternative splicing events fall into three major categories: isoform-based, exon-based, and event-based methods. Isoform based methods first assembly the full-length transcripts and then estimate their relative abundances. Exon-based methods detect alternative splicing by comparing the distribution of reads on exon and junctions between comparisons. This methods only detects the differentially expressed exons/junctions and not the type of event. In event-based methods, splicing events themselves are quantified using the fraction of mRNAs that contains a specific form of that event. Exon-based methods have a greater accuracy to detect splicing events and are appropriate if the focus is on the event themself and not the study of the whole isoform.</p><h1 class="h2 pt-4">Workflow</h1><div class="container-lg page-section pt-2"><div class="container-lg d-flex flex-column flex-sm-row flex-sm-justify-between flex-sm-items-center py-3 px-3"><div class="d-flex flex-self-center" style="flex-basis:10%"><h3 class="h3 text-center mb-auto">Step 1</h3></div><div class="d-flex flex-column" style="flex-basis:90%"><h2 class="h3">Quality control</h2><p class="f3">The first step in any RNA-Seq workflow is to assess the quality of the sequenced reads. A first quality control (QC) step is recommended for the assessement of contaminations, bias and errors in raw sequences. By addressing these issues we can reduce the impact of these errors in downstream analysis.</p></div></div><div class="container-lg d-flex flex-column flex-sm-row flex-sm-justify-between flex-sm-items-center py-3 px-3"><div class="d-flex flex-self-center" style="flex-basis:10%"><h3 class="h3 text-center mb-auto">Step 2</h3></div><div class="d-flex flex-column" style="flex-basis:90%"><h2 class="h3">Read Alignment</h2><p class="f3">In this step, reads are aligned to either the species reference genome or transcriptome using a splicing-aware aligner sofware.</p></div></div><div class="container-lg d-flex flex-column flex-sm-row flex-sm-justify-between flex-sm-items-center py-3 px-3"><div class="d-flex flex-self-center" style="flex-basis:10%"><h3 class="h3 text-center mb-auto">Step 3</h3></div><div class="d-flex flex-column" style="flex-basis:90%"><h2 class="h3">Quantification of transcript expression</h2><p class="f3">After the alignment step, reads are assigned to a gene or a transcrip which is then used to estimate expression levels of that gene/transcript.</p></div></div><div class="container-lg d-flex flex-column flex-sm-row flex-sm-justify-between flex-sm-items-center py-3 px-3"><div class="d-flex flex-self-center" style="flex-basis:10%"><h3 class="h3 text-center mb-auto">Step 4</h3></div><div class="d-flex flex-column" style="flex-basis:90%"><h2 class="h3">Quality assessment</h2><p class="f3">In the last step, an additional quality control step assessing the percentage of reads mapped to the transcriptome versus unmapped will be conducted. This step can when detect contaminations on the extraction or library construction steps which can impair the correct analysis of the data.</p></div></div></div><div class="container-lg d-flex flex-column justify-content-evenly pt-4"><h1 class="h2">Available pipelines</h1><div class="container-lg page-section py-2"><div class="d-md-flex flex-wrap gutter flex-auto"></div></div></div></div></div></body></html>