---
layout: default
title: Variant analysis

eleventyNavigation:
  key: Single-cell expression analysis
  parent: Transcriptomics
  order: 5
---

<script> function mouseOver(x) {document.getElementById("legend").innerHTML = x.title;}</script>

<div class="color-bg-tertiary-light">
  <div class="container-lg text-center" >
    <img
      alt=""
      src="/assets/img/scrna_seq_analysis.png"
      class="block width-fit"
      title="Variant analysis"
      />
  </div>
  
  <div class="container-lg d-flex flex-column justify-content-evenly py-3 px-3">
    <style> div {text-align: justify; text-justify: inter-word; } </style> 
    
    <h1 class="h2">Single-cell expression analysis (scRNA-seq)</h1>
    <p class="f3">
      Transcriptional analysis at the single-cell level has become the technology of choice to assess cellular heterogeneity in tissues and for elucidating the interplay dynamics between cellular populations. At the technical level, there are two main paradigm in scRNA-seq which in turn influence the analysis step. The most common approach is to use microscopic droplets or wells to isolate a large number of cells and then sequence the libraries relatively shallow. This high-througput, low-depth strategy (typical 10x chromium platform) has the advantage to support unique molecular identifiers (UMIs). UMIs are short barcodes that are attached to transcripts before amplification, allowing the removal of PCR duplicates and obtain a more accurate estimate of expression levels. The major shortcoming is that it only allows for the 5′ or 3′ end of each mRNA to be sequences. The other scRNA-seq paradigm takes an low-throughput, high-depth approach, isolating cells into individual wells and apply the Smart-seq2 protocol. This method show higher sensitivity than droplet based technologies and allow for the entire transcript to be profiled.
    </p>

    <h1 class="h2 pt-4">Workflow</h1>
    <div class="container-lg page-section pt-2">
      <div class="container-lg d-flex flex-column flex-sm-row flex-sm-justify-between flex-sm-items-center py-3 px-3">
	<div class="d-flex flex-self-center" style="flex-basis:10%">
	  <h3 class="h3 text-center mb-auto">Step 1</h3>
	</div>
	
	<div class="d-flex flex-column" style="flex-basis:90%" >
	  <h2 class="h3">Demultiplexing, quality control and filtering</h2>
	  <p class="f3">
	    We will first start by demultiplexing reads belonging to the same cell followed by a quality control step where low quality reads, adapters and primers will be removed from reads.
	  </p>
	</div>
      </div>

      <div class="container-lg d-flex flex-column flex-sm-row flex-sm-justify-between flex-sm-items-center py-3 px-3">
	<div class="d-flex flex-self-center" style="flex-basis:10%">
	  <h3 class="h3 text-center mb-auto">Step 2</h3>
	</div>
	
	<div class="d-flex flex-column" style="flex-basis:90%" >
	  <h2 class="h3">Alignment and detection of transcripts</h2>
	  <p class="f3">
	    After the initial processing and QC, alignment to the genome or transcriptome is performed in order to detect and quantify the transcripts in each cell. After alignment, expression levels need will be quantified by counting the reads aligned to each gene.  
	  </p>
	</div>
      </div>

      <div class="container-lg d-flex flex-column flex-sm-row flex-sm-justify-between flex-sm-items-center py-3 px-3">
	<div class="d-flex flex-self-center" style="flex-basis:10%">
	  <h3 class="h3 text-center mb-auto">Step 3</h3>
	</div>
	
	<div class="d-flex flex-column" style="flex-basis:90%" >
	  <h2 class="h3">Filtering</h2>
	  <p class="f3">
	    Following the alignment and quantification step, several filtering steps are necessary to remove low-quality data or outliers. Per-sample metrics such as the number of reads mapped to each cell, the number of genes or trancripts detected and the percentage of reads mapping to mitochondrial gens (increased mitochondrial RNA content is usually associated with dead cells) or spike-ins are useful to measure sample quality. Data originating from doublets or aggregates of more cells will be discarded.
	  </p>  
	</div>
      </div>
    </div>
  
    <div class="container-lg d-flex flex-column justify-content-evenly pt-4">
      <h1 class="h2">Available pipelines</h1>
      <div class="container-lg page-section py-2">
	<div class="d-md-flex flex-wrap gutter flex-auto">
	  {% assign apps = collections.genomics %}
	  {% for app in apps %}{% if 'dna_read_assembly' == app.data.label %}
	  {% include "genomic" app: app.data %}{% endif %}
	  {% endfor %}
	</div>
      </div>
    </div>

  
    <h1 class="h2 pt-4">Follow-up analysis</h1>
    <div class="container-lg page-section py-2">
      <div class="d-md-flex flex-wrap gutter flex-auto">
	<div class="d-flex flex-column" style="flex-basis:33.33%">
	  <div class="d-flex flex-self-center anim-hover-grow" >
	    <a href="/analysis/transcriptomics/visual_representations/principal_component_analysis/">
	      <img
		alt=""
		src="/assets/img/PCA.png"
		onmouseover="mouseOver(this)"
		class="block width-fit"
		title="Small variant analysis"
		/>
	    </a>
	  </div>
	  <div class="page-section d-flex flex-column mt-auto">
	    <h3 class="h3 text-center mb-auto">Principal component analysis</h3>    
	  </div>
	</div>
	
	<div class="d-flex flex-column" style="flex-basis:33.33%">
	  <div class="d-flex flex-self-center anim-hover-grow" >
	    <a href="/analysis/transcriptomics/single_cell_expression_analysis/cell_type_identification/">
	      <img
		alt=""
		src="/assets/img/cellTypeIdentification.png"
		onmouseover="mouseOver(this)"
		class="block width-fit"
		title="Small variant analysis"
		/>
	    </a>
	  </div>
	  <div class="page-section d-flex flex-column mt-auto">
	    <h3 class="h3 text-center mb-auto">Cell-type identification</h3>    
	  </div>
	</div>
	
	<div class="d-flex flex-column" style="flex-basis:33.33%">
	  <div class="d-flex flex-self-center anim-hover-grow">
	    <a href="/analysis/transcriptomics/single_cell_expression_analysis/trajectory_analysis/">
	      <img
		alt=""
		src="/assets/img/TrajectoryAnalysis.png"
		onmouseover="mouseOver(this)"
		class="d-block width-fit"
		title="Copy-number analysis"
		/>
	    </a>
	  </div>
	  
	  <div class="page-section d-flex flex-column mt-auto">
	    <h3 class="h3 text-center mb-auto">Trajectory analysis</h3>    
	  </div>
	</div>
	
	<div class="d-flex flex-column" style="flex-basis:33.33%">
	  <div class="d-flex flex-self-center anim-hover-grow">
	    <a href="/analysis/transcriptomics/exploratory_analysis/differentia_expression_analysis/">
	      <img
		alt=""
		src="/assets/img/DEanalysis.png"
		onmouseover="mouseOver(this)"
		class="d-block width-fit"
		title="Copy-number analysis"
		/>
	    </a>
	  </div>
	  
	  <div class="page-section d-flex flex-column mt-auto">
	    <h3 class="h3 text-center mb-auto">Differential expression analysis</h3>    
	  </div>
	</div>
	
 	<div class="d-flex flex-column" style="flex-basis:33.33%">
	  <div class="d-flex flex-self-center anim-hover-grow" >
	    <a href="/analysis/genomics/variant_detection/structural_variants/">
	      <img
		alt=""
		src="/assets/img/pathwayAnalysis.png"
		onmouseover="mouseOver(this)"
		class="d-block width-fit"
		title="Structural variant analysis"
		/>
	    </a>
	  </div>
	  <div class="page-section d-flex flex-column mt-auto">
	    <h3 class="h3 text-center mb-auto">Pathway analysis</h3>
	  </div>
	</div>
	
	<div class="d-flex flex-column" style="flex-basis:33.33%">
	  <div class="d-flex flex-self-center anim-hover-grow" >
	    <a href="/analysis/genomics/variant_detection/structural_variants/">
	      <img
		alt=""
		src="/assets/img/predictiveModelling.png"
		onmouseover="mouseOver(this)"
		class="d-block width-fit"
		title="Structural variant analysis"
		/>
	    </a>
	  </div>
	  <div class="page-section d-flex flex-column mt-auto">
	    <h3 class="h3 text-center mb-auto">Predictive modeling</h3>
	  </div>
	</div>
      </div>
    </div>
  </div>
</div>
