<!doctype html><html lang="en" class="height-full" data-color-mode="auto" data-light-theme="light" data-dark-theme="dark"><head><meta charset="utf-8"><meta name="viewport" content="width=device-width,initial-scale=1"><link rel="stylesheet" href="/assets/dist/styles.css"><link rel="apple-touch-icon" sizes="180x180" href="/apple-touch-icon.png?v=jw3d9vxKo4"><link rel="icon" type="image/png" sizes="32x32" href="/favicon-32x32.png"><link rel="icon" type="image/png" sizes="16x16" href="/favicon-16x16.png"><link rel="manifest" href="/site.webmanifest?v=jw3d9vxKo4"><link rel="mask-icon" href="/safari-pinned-tab.svg?v=jw3d9vxKo4" color="#5bbad5"><link rel="shortcut icon" href="/i3s.ico"><meta name="theme-color" content="#66cce0"></head><body><div class=""><div class="container-lg d-flex flex-items-center flex-justify-between px-3"><div class="py-3 d-inline-flex flex-items-center"><div class="col-1"><a href="/"><img src="/assets/img/i3s.jpg" class="width-fit" alt="i3S logo"></a></div><div class="text-inherit text-left"><h1 class="h4 text-uppercase">Bioinformatic Workflows</h1></div></div><details class="dropdown details-reset details-overlay d-inline-block focus-within"><summary class="h1" aria-haspopup="true">â‰¡</summary><div class="dropdown-menu dropdown-menu-w"><li><details class="dropdown details-reset details-overlay"><summary class="" aria-haspopup="true"><p class="dropdown-item" style="margin-bottom:0">Genomics</p></summary><div class="dropdown-menu dropdown-menu-w px-3"><li class=""><a class="dropdown-item" href="/analysis/genomics/de_novo_sequencing2/">Genome assembly</a></li><li><a class="dropdown-item" href="/analysis/genomics/variant_detection/">Variant analysis</a></li><li><a class="dropdown-item" href="/analysis/genomics/metagenomic_analysis/">Metagenomics</a></li><li><a class="dropdown-item" href="/analysis/genomics/genetic_analysis/">Genetic analysis</a></li></div></details></li><li><details class="dropdown details-reset details-overlay"><summary class="" aria-haspopup="true"><p class="dropdown-item" style="margin-bottom:0">Transcriptomics</p></summary><div class="dropdown-menu dropdown-menu-w px-3"><li><a class="dropdown-item" href="/analysis/transcriptomics/exploratory_analysis/">Exploratory gene expression analysis</a></li><li><a class="dropdown-item" href="/analysis/transcriptomics/rna_seq_differential_expression_analysis/">RNA-seq analysis</a></li><li><a class="dropdown-item" href="/analysis/transcriptomics/pathway_analysis/">Pathway analysis</a></li><li><a class="dropdown-item" href="/analysis/transcriptomics/transcriptome_assembly/">Transcriptome assembly</a></li><li><a class="dropdown-item" href="/analysis/transcriptomics/single_cell_expression_analysis/">Single-cell expression analysis</a></li><li><a class="dropdown-item" href="/analysis/transcriptomics/microRNA_data_analysis/">MicroRNA data analysis</a></li><li><a class="dropdown-item" href="/analysis/transcriptomics/alternative_splicing_analysis/">Alternative splicing analysis</a></li><li><a class="dropdown-item" href="/analysis/transcriptomics/fusion_gene_detection/">Fusion gene detection</a></li></div></details></li></div></details></div></div><script>function mouseOver(x) {document.getElementById("legend").innerHTML = x.title;}</script><div class="color-bg-tertiary-light"><div class="container-md" style="flex-basis:66.66%"><img alt="" src="/assets/img/assembly.png" onmouseover="mouseOver(this)" class="block width-fit" title="Exploratory analysis"></div><div class="container-lg d-flex flex-column justify-content-evenly py-3 px-3"><style>div {text-align: justify; text-justify: inter-word; }</style><h1 class="h2">Genome assembly and annotation</h1><p class="f3">The goal of de novo genome assembly is to determine the sequence of a genome using only randomly sampled sequence fragments. This technique is usually employed for organosms with no reference genome or highly dynamic genomes. In de novo assembly, random fragments from a genome are sequenced and computationally stitched together to generate sequence contigs and scaffolds. The key aspects for achieving high quality de novo assemblies, particularly on highly repetitive genomes, are read length and the assembly algorithm. An assembled genome can then be annotated based on sequence homology, predicted gene sequences and, if available, RNA-sequencing data from the same organism. If annotated genomes for close relative species exist, the annotation can be improved by transferring gene information to the newly assembled genome. The quality of an assembled genome is assessed using metrics such as N50, L50 and completness with regards to highly conserved orthologs.</p><h1 class="h2 pt-4">Workflow</h1><div class="container-lg page-section pt-2"><div class="container-lg d-flex flex-column flex-sm-row flex-sm-justify-between flex-sm-items-center py-3 px-3"><div class="d-flex flex-self-center" style="flex-basis:10%"><h3 class="h3 text-center mb-auto">Step 1</h3></div><div class="d-flex flex-column" style="flex-basis:90%"><h2 class="h3">DNAseq read assembly</h2><p class="f3">Due to the high variablility between de novo assemblers, a conjugation of the following workflows is recommended in order to optimize contig length (e.g. N50), genome coverage and maximum, median and average contig size.</p></div></div><div class="container-lg d-flex flex-column flex-sm-row flex-sm-justify-between flex-sm-items-center py-3 px-3"><div class="d-flex flex-self-center" style="flex-basis:10%"><h3 class="h3 text-center mb-auto">Step 2</h3></div><div class="d-flex flex-column" style="flex-basis:90%"><h2 class="h3">Repeat identification and masking</h2><p class="f3">Repeat identification and masking is usually a previous step to the gene prediction and annotation phase. The term 'masking' means transforming every nucleotide identified as a repeat to an 'N', 'X' or to a lower case a, t, g, or c (the latter is known as soft masking). The masking step signals to downstream sequence alignment and gene prediction tools that these regions are repeats. Identifying repeats is complicated by the fact that repeats are often poorly conserved; thus, accurate repeat detection usually requires a repeat library for the species of interest.</p></div></div><div class="container-lg d-flex flex-column flex-sm-row flex-sm-justify-between flex-sm-items-center py-3 px-3"><div class="d-flex flex-self-center" style="flex-basis:10%"><h3 class="h3 text-center mb-auto">Step 3</h3></div><div class="d-flex flex-column" style="flex-basis:90%"><h2 class="h3">Assembly quality control and completeness</h2><p class="f3">A good metric of assembly quality is evaluating the completeness. This consists of assessing the extent to which the assembly recovers single-copy orthologs that are present across higher taxonomic groups. The expectations for these genes to be found in a genome/transcriptome in single-copy are evolutionarily strong. If they cannot be identified, it is possible that the sequencing and/or assembly have failed to capture the complete expected gene content.</p></div></div><div class="container-lg d-flex flex-column flex-sm-row flex-sm-justify-between flex-sm-items-center py-3 px-3"><div class="d-flex flex-self-center" style="flex-basis:10%"><h3 class="h3 text-center mb-auto">Step 4</h3></div><div class="d-flex flex-column" style="flex-basis:90%"><h2 class="h3">Annotation</h2><p class="f3">Annotation of the assembly genome can be divided into structural annotation and functional annotation. In structural annotation, gene prediction tools are used to locate genes within the scaffolds, and the structures of introns, exons, and untranslated regions (UTRs) constituting these genes. In the functional annotation step, homology search and ontology mapping are performed using structure-annotated sequences in order to determine the functions of the genes.</p></div></div></div><div class="container-lg d-flex flex-column justify-content-evenly px-3 pt-4"><h1 class="h2">Available pipelines</h1><div class="container-lg page-section py-2"><div class="d-md-flex flex-wrap gutter flex-auto"></div></div></div><div class="container-lg d-flex flex-column justify-content-evenly px-3"><h1 class="h2">Follow-up analysis</h1><div class="container-lg page-section pt-2"><div class="container-lg d-flex flex-column flex-sm-row flex-sm-justify-between flex-sm-items-center py-3 px-3"><div class="d-flex flex-self-center" style="flex-basis:30%"><h3 class="h3 mb-auto">Visual representations</h3></div><div class="d-flex anim-hover-grow" style="flex-basis:60%"><a href="/analysis/genomics/de_novo_sequencing2/"><img alt="" src="/assets/img/circos.png" onmouseover="mouseOver(this)" class="block" style="width:100px;" title="Circos plot"></a></div></div><div class="container-lg d-flex flex-column flex-sm-row flex-sm-justify-between flex-sm-items-center py-3 px-3"><div class="d-flex flex-self-center" style="flex-basis:30%"><h3 class="h3 mb-auto">Phylogenetic analysis and comparative genomics</h3></div><div class="d-flex anim-hover-grow" style="flex-basis:60%"><a href="/analysis/genomics/de_novo_sequencing2/"><img alt="" src="/assets/img/phylogenetic_analysis.png" onmouseover="mouseOver(this)" class="block" style="width:100px;" title="Phylogenetic analysis"></a></div></div></div></div></div></div></body></html>