---
layout: default
title: Variant analysis

eleventyNavigation:
  key: MicroRNA data analysis
  parent: Transcriptomics
  order: 6
---

<script> function mouseOver(x) {document.getElementById("legend").innerHTML = x.title;}</script>

<div class="color-bg-tertiary-light">
  <div class="container-lg text-center" >
    <img
      alt=""
      src="/assets/img/rnaseq/mirna_seq_analysis.png"
      class="block"
      title="microRNA data analysis"
      style="max-width: 300px;"
      />
  </div>
  
  <div class="container-lg d-flex flex-column justify-content-evenly py-3 px-3">
    <style> div {text-align: justify; text-justify: inter-word; } </style> 
    
    <h1 class="h2">MicroRNA data analysis</h1>
    <p class="f3">
      MicroRNA (miRNAs) are a subtype of small RNAs, averaging 22 nucleotides, which post-transcriptionally regulate gene expression in a broad range of organisms. Although miRNA can be extracted from total RNA-seq, technical challanges (especially on the amplification step of library construction) can lead to inaccurate quantification of miRNAs, particularly on samples with low starting material, such as the ones extracted from serum. For this cases and for miRNA biomarker discovery, miRNA sequencing (miRNA-seq) is the recommened strategy to accurately target miRNA. Although the bioinformatic analysis is the same regadless of the sequencing strategy, miRNA library kits that take advantage of unique moleculare indices (UMI) to takle PCR bias, require an additional cleaning step.  

      
    </p>
     <h1 class="h2 pt-4">Workflow</h1>
    <div class="container-lg page-section pt-2">
      <div class="container-lg d-flex flex-column flex-sm-row flex-sm-justify-between flex-sm-items-center py-3 px-3">
	<div class="d-flex flex-self-center" style="flex-basis:10%">
	  <h3 class="h3 text-center mb-auto">Step 1</h3>
	</div>
	
	<div class="d-flex flex-column" style="flex-basis:90%" >
	  <h2 class="h3">Demultiplexing, quality control and filtering</h2>
	  <p class="f3">
	    Due to the short size of reads, an extensive quality control step is require to ensure correct alignments. Adapter artifacts and low quality sequences bases will be excluded from the analysis which will be followed by a read length exclusion threshold. Reads with length inferior to 16 nucleotides, representing degraded RNA, and superior to 30 nucleotides, will be removed. After this pre-processing step, read lengths should exhibit a distint peak between 18-23 nucleotides.
	  </p>
	</div>
      </div>

      <div class="container-lg d-flex flex-column flex-sm-row flex-sm-justify-between flex-sm-items-center py-3 px-3">
	<div class="d-flex flex-self-center" style="flex-basis:10%">
	  <h3 class="h3 text-center mb-auto">Step 2</h3>
	</div>
	
	<div class="d-flex flex-column" style="flex-basis:90%" >
	  <h2 class="h3">Alignment</h2>
	  <p class="f3">
	    The alignment step will be carried with stringent parameters, comparatively to those of transcriptome alignment, due to the small size of reads. For accurate detection and to reduce the level of false positives, two rounds of alignment will be performed, one against mature miRNA sequences from miRBase database of the organisms in analysis and a second against the species reference genome.
	  </p>
	</div>
      </div>

      <div class="container-lg d-flex flex-column flex-sm-row flex-sm-justify-between flex-sm-items-center py-3 px-3">
	<div class="d-flex flex-self-center" style="flex-basis:10%">
	  <h3 class="h3 text-center mb-auto">Step 3</h3>
	</div>
	
	<div class="d-flex flex-column" style="flex-basis:90%" >
	  <h2 class="h3">miRNA quantification</h2>
	  <p class="f3">
	    After alignment, quantification of miRNA will be caried by counting reads to each miRNA.
	  </p>  
	</div>
      </div>
    </div>
  
    <div class="container-lg d-flex flex-column justify-content-evenly pt-4">
      <h1 class="h2">Available pipelines</h1>
      <div class="container-lg page-section py-2">
	<div class="d-md-flex flex-wrap gutter flex-auto">
	  {% assign apps = collections.genomics %}
	  {% for app in apps %}{% if 'dna_read_assembly' == app.data.label %}
	  {% include "genomic" app: app.data %}{% endif %}
	  {% endfor %}
	</div>
      </div>
    </div>
  </div>
</div>
