---
layout: default
title: Genomic analysis
---

<script> function mouseOver(x) {document.getElementById("legend").innerHTML = x.title; } </script>

<div class="color-bg-tertiary-light">
  <div class="container-lg d-flex flex-column justify-content-evenly py-3 px-3">
    <style> div {text-align: justify; text-justify: inter-word; } </style>
    <p class="lead text-justify">
      <strong>Disclamer:</strong> The information on this page aims to highlight the array of bioinformatic analysis that can be performed at the i3S bioinformatic service. Given the array of fields at i3S, the background provided in this page will not be exaustive. When possible references to published work will be provided. If you require a specific genomic bioinformatic analysis not present in this page, please contact the i3S bioinformatic service.
    </p>

    <h1 class="h2">DNA sequencing data analysis</h1>
    
    <div class="container-lg d-flex flex-column flex-sm-row flex-sm-justify-between flex-sm-items-center py-3 px-3">
      <div class="d-flex flex-self-center" style="flex-basis:33.33%">
	<h3 class="h3 text-center mb-auto">Genome assembly</h3>
      </div>
      
      <div class="d-flex anim-hover-grow" style="flex-basis:66.66%" >
	<a href="/analysis/genomics/de_novo_sequencing2/">
	  <img
	    alt=""
	    src="/assets/img/genome_assembly.png"
	    onmouseover="mouseOver(this)"
	    class="block width-fit"
	    title="Genome assembly"
            />
	</a>
      </div>
    </div>

    <div class="container-lg d-flex flex-column flex-sm-row flex-sm-justify-between flex-sm-items-center py-6 px-3">
      <div class="d-flex flex-self-center" style="flex-basis:33.33%">
	<h3 class="h3 text-center mb-auto">Variant analysis</h3>
      </div>
      
      <div class="d-flex anim-hover-grow" style="flex-basis:66.66%" >
	<a href="/analysis/genomics/variant_detection/">
	  <img
	    alt=""
	    src="/assets/img/variantAnalysis.png"
	    onmouseover="mouseOver(this)"
	    class="block width-fit"
	    title="Variant analysis"
            />
	</a>
      </div>
    </div>

    <div class="container-lg d-flex flex-column flex-sm-row flex-sm-justify-between flex-sm-items-center py-6 px-3">
      <div class="d-flex flex-self-center" style="flex-basis:33.33%">
	<h3 class="h3 text-center mb-auto">Metagenomics</h3>
      </div>
      
      <div class="d-flex anim-hover-grow" style="flex-basis:66.66%" >
	<a href="/analysis/genomics/variant_detection/">
	  <img
	    alt=""
	    src="/assets/img/metagenomics.png"
	    onmouseover="mouseOver(this)"
	    class="block width-fit"
	    title="Variant analysis"
            />
	</a>
      </div>
    </div>

     <div class="container-lg d-flex flex-column flex-sm-row flex-sm-justify-between flex-sm-items-center py-6 px-3">
      <div class="d-flex flex-self-center" style="flex-basis:33.33%">
	<h3 class="h3 text-center mb-auto">Genetic analysis</h3>
      </div>
      
      <div class="d-flex anim-hover-grow" style="flex-basis:66.66%" >
	<a href="/analysis/genomics/variant_detection/">
	  <img
	    alt=""
	    src="/assets/img/geneticAnalysis.png"
	    onmouseover="mouseOver(this)"
	    class="block width-fit"
	    title="Variant analysis"
            />
	</a>
      </div>
     </div>

    
    <h1 class="h2">Background</h1>
    <p class="f3">

    </p>
    <h1 class="h2">1. Alignment strategy</h1>
    
    <h2 class="p-1 h3">
      1.1 De Novo Sequencing
      <a href="/analysis/genomics/de_novo_sequencing/">
	<img
	  alt=""
          src="/assets/img/analysis.png"
          class="avatar avatar-1 mr-2 color-bg-transparent"
	  style="box-shadow: none;"
          />
      </a>
    </h2>
    
    <p class="p-2 f3">
      The goal of de novo genome assembly is to determine the sequence of a genome using only randomly sampled sequence fragments. This technique is usually employed on novel genomes when a reference or template sequence is not available. In de novo assembly, random fragments from a genome are sequenced and computationally stitched together to generate sequence contigs and scaffolds. The key aspects for achieving high quality de novo assemblies, particularly on highly repetitive genomes, are read length, overlap mapping quality and assembly algorithm. High quality genome assemblies usually employ both short-read and long-read genome sequencing stratagies. 
    </p>
    
    <h2 class="p-1 h3">1.2 Resequencing
      <a href="/analysis/genomics/resequencing/">
	<img
	  alt=""
          src="/assets/img/analysis.png"
          class="avatar avatar-1 mr-2 color-bg-transparent"
	  style="box-shadow: none;"
          />
      </a>
    </h2>
    <p class="p-2 f3">
      Resequencing is typically performed when a reference genome sequence is available. Sequencing reads are aligned back to the reference to determine the location in the genome the specific read best matches. Resequencing is often applied to variant detection (single nucleotide polymorphisms, small insertions/deletions, structural variants, copy number variation) and derivatives such as tumor vs normal comparison, population genetic analysis, Mendelian disease analysis, and trio sequencing.
    </p>
    <div class="p-0 mr-3 Box color-fg-muted d-inline-block f4">
      <h2 class="p-2 h4">Q1. Which sequencing platform should I choose?</h2>
      <p class="px-2">
	The answer is highly dependent on many factors, including the goals of your experiment, the organism of interest, and the amount of material available. The table below summarizes the most common sequencing platforms, their main applications and advantages. Sequencing platforms with higher number of reads per run have a lower cost per gigabase which is advantageous for large genomes and/or when multiplexing a large number of samples.
      </p>
      <div class="px-2">
	<style type="text/css">table.tableizer-table {width: 100%; font-size: 12px; border: 1px solid #CCC;} .tableizer-table td {padding: 4px; margin: 3px; border: 1px solid #CCC;} .tableizer-table tc { paddi
																					     ng: 4px; margin: 3px;} .tableizer-table th { padding: 4px; background-color: #104E8B; color: #FFF; font-weight: bold;}</style>

      <p class="">Short-read platforms</p>
      <table class="tableizer-table text-right">
	<thead><tr class="tableizer-firstrow">
	    <th>Platform</th>
	    <th>Company</th>
	    <th>Read length (bp)</th>
	    <th>Max Yield (Gb)</th>
	    <th>In i3S</th>
	    <th>Applications</th></tr>
	</thead>
	<tbody>
	  <tr><td>MiSeq</td><td>Illumina</td><td>250</td><td>15</td><td>No</td><td>Small genome, targeted sequencing, metagenomics</td></tr>
	  <tr><td>NextSeq 550</td><td>Illumina</td><td>150</td><td>120</td><td>Yes</td><td>Small genome, targeted sequencing, metagenomics, WES, WGS</td></tr>
	  <tr><td>HiSeq 2500</td><td>Illumina</td><td>125</td><td>9-1000</td><td>No</td><td>WGS, WES</td></tr>
	  <tr><td>HiSeq 4000</td><td>Illumina</td><td>150</td><td>105-1500</td><td>No</td><td>WES, WGS</td></tr>
	  <tr><td>HiSeq X10</td><td>Illumina</td><td>150</td><td>1600-1800</td><td>No</td><td>WES, WGS</td></tr>
	  <tr><td>NovaSeq 6000</td><td>Illumina</td><td>250</td><td>6000</td><td>No</td><td>WES, WGS</td></tr>
	  <tr><td>Ion Proton</td><td>Thermofisher</td><td>200</td><td>15</td><td>No</td><td>Targeted panels, WES, small genomes </td></tr>
	  <tr><td>Ion S5</td><td>Thermofisher</td><td>200</td><td>15</td><td>Yes</td><td>Targeted panels, WES, small genomes, metagenomics </td></tr>
      </tbody></table>

      <p class="pt-3">Long-read platforms</p>
      <table class="tableizer-table text-right">
	<thead><tr class="tableizer-firstrow"><th>Platform</th><th>Company</th><th>Read length (bp)</th><th>Max Yield (Gb)</th><th>In i3S</th><th>Applications</th></tr></thead><tbody>
	  <tr><td>PacBio CCS</td><td>PacBio</td><td>11000</td><td>2</td><td>No</td><td>De novo assembly, SV detection, gene isoform resolution, epigenetic modification </td></tr>
	  <tr><td>PromethION</td><td>Oxford Nanopore</td><td>8000</td><td>20-130</td><td>No</td><td>Population-based studies. De novo assembly, SV detection, metagenomics </td></tr>
	  <tr><td>MinION</td><td>Oxford Nanopore</td><td>3500</td><td>50</td><td>No</td><td>De novo assembly, SV detection, metagenomics </td></tr>
      </tbody></table>
      </div>
      
      <h2 class="p-2 py-2 h4">Q2. How much coverage do I need?</h2>
      <p class="px-2">
	The answer is highly dependent on many factors, including the goals of your experiment, the organism of interest, and the amount of material available. Below are some generalized recommendations:
      </p>

      <div class="container-md clearfix d-flex d-flex-column">
	<div class="col-3 float-left pr-2 flex-self-center">
	  <div class="text-center">Short-read genome sequencing</div>
	</div>
	<div class="col-9 float-left pl-2">
	  <div class="text-left">
	    <ul class="list-style-none">
	      <li>Germline/frequent variant analysis: 20-50x</li>
	      <li>Somatic/rare variants: 100-1000x</li>
	      <li>Tumor vs Normal: ≥100x tumor, ≥30x normal</li>
	      <li>De novo assembly: 100-1000x</li>
	    </ul>
	  </div>
	</div>
      </div>

      <div class="container-md clearfix d-flex d-flex-column py-3">
	<div class="col-3 float-left pr-2 flex-self-center">
	  <div class="text-center">Long-read genome sequencing:</div>
	</div>
	<div class="col-9 float-left pl-2">
	  <div class="text-left">
	    <ul class="list-style-none">
	      <li>Gap filling and scaffolding: 10x</li>
	      <li>Large structural variant detection: 10x</li>
	      <li>Germline/frequent variant analysis: 20-50x</li>
	      <li>De novo assembly: 50-100x</li>
	    </ul>
	  </div>
	</div>
      </div>

      <h2 class="p-2 h4">Q3. How do I calculate coverage?</h2>
      <p class="px-2">
	Coverage is a multiplier based on the total size of the genome (see below). For humans, 30x coverage can be achieved with 600 million reads of 150 bp (or 300M paired-end reads). Please note that nowadays companies advertise coverage as the number of gigabases given, meaning that 600M reads of 150bp corresponde to 90Gbp (600 x 10<sup>6</sup> reads x 150 bp/read / 10<sup>9</sup>). By estimating based on gigabases instead of coverage the company is protecting themselves against the fact that not all reads sequenced will align against the target genome. 
      	<ul class="pl-6 list-style-none">
	  <li>Coverage = (read length) x (total number of reads) / (genome size)</li>
	  <li> Example: 30x = (150 bp/read) x (600 x 10<sup>6</sup> reads) / (3 x 10<sup>9</sup> bp)</li>
	</ul>
      </p>
    </div>
    
    <h2 class="p-1 h3 py-2">1.3 Genotyping by microarray</h2>
    <p class="p-2 f3">
       Microarrays genotyping is a common and cost-efficient strategy to detect single nucleotide polymorphisms (SNPs) or larger structural changes of known markers in a species genome. These microarrays can containg million of markers and allow for customization of the array content and are commonly used for high-throughput, multiplex processing for large-scale population studies, clinical research. Although there are microarray solutions for non-human genomes, human microarrays are the most predominant. 
    </p>

    <h1 class="h2">2. Variant detection</h1>
    <h2 class="p-1 h3">2.1 Single Nucleotide Polymorphisms (SNPs)</h2>
    <p class="p-2 f3">
       
    </p>
    <h2 class="p-1 h3">2.2 Insertions/deletions</h2>
    <p class="p-2 f3">
       
    </p>
    <h2 class="p-1 h3">2.3 Structural variants</h2>
    <p class="p-2 f3">
       
    </p>
    <h2 class="p-1 h3">2.4 Copy number variation</h2>
    <p class="p-2 f3">
       
    </p>

    <h1 class="h2 py-2">3. Variant annotation</h1>
    <h2 class="p-1 h3">3.1. Annotation using a reference sequence</h2>
    <p class="p-2 f3">
       
    </p>
    <h2 class="p-1 h3">3.2. Reference-free annotation</h2>
    <p class="p-2 f3">
       
    </p>
    <h1 class="h2 py-2">4. Visualization</h1>
    <h2 class="p-1 h3">4.1. </h2>
    <p class="p-2 f3">
       
    </p>
    
  </div>
  
  <!-- div class="container-lg px-3 page-section">
    <h1 class="h2">Available Workflows</h1>
    <h1 class="h2">{{ page.title }}</h1>
    
    <div class="d-md-flex flex-wrap gutter flex-auto">
      {% assign apps = collections.genomics %}
      {% for app in apps %}
      {% include "genomic" app: app.data %}
      {% endfor %}
    </div>
    -->
  </div>
</div>

