<!doctype html><html lang="en" class="height-full" data-color-mode="auto" data-light-theme="light" data-dark-theme="dark"><head><meta charset="utf-8"><meta name="viewport" content="width=device-width,initial-scale=1"><link rel="stylesheet" href="/assets/dist/styles.css"><link rel="apple-touch-icon" sizes="180x180" href="/apple-touch-icon.png?v=jw3d9vxKo4"><link rel="icon" type="image/png" sizes="32x32" href="/favicon-32x32.png"><link rel="icon" type="image/png" sizes="16x16" href="/favicon-16x16.png"><link rel="manifest" href="/site.webmanifest?v=jw3d9vxKo4"><link rel="mask-icon" href="/safari-pinned-tab.svg?v=jw3d9vxKo4" color="#5bbad5"><link rel="shortcut icon" href="/i3s.ico"><meta name="theme-color" content="#66cce0"></head><body><div class=""><div class="container-lg d-flex flex-items-center flex-justify-between px-3"><div class="py-3 d-inline-flex flex-items-center"><div class="col-1"><a href="/"><img src="/assets/img/i3s.jpg" class="width-fit" alt="i3S logo"></a></div><div class="text-inherit text-left"><h1 class="h4 text-uppercase">Bioinformatic Workflows</h1></div></div><details class="dropdown details-reset details-overlay d-inline-block focus-within"><summary class="h1" aria-haspopup="true">â‰¡</summary><div class="dropdown-menu dropdown-menu-w"><li><details class="dropdown details-reset details-overlay"><summary class="" aria-haspopup="true"><p class="dropdown-item" style="margin-bottom:0">Genomics</p></summary><div class="dropdown-menu dropdown-menu-w px-3"><li><a class="dropdown-item" href="/analysis/genomics/de_novo_sequencing2/">Genome assembly</a></li><li><a class="dropdown-item" href="/analysis/genomics/variant_detection/">Variant analysis</a></li><li><a class="dropdown-item" href="/analysis/genomics/metagenomic_analysis/">Metagenomics</a></li><li><a class="dropdown-item" href="/analysis/genomics/genetic_analysis/">Genetic analysis</a></li></div></details></li><li><details class="dropdown details-reset details-overlay"><summary class="" aria-haspopup="true"><p class="dropdown-item" style="margin-bottom:0">Transcriptomics</p></summary><div class="dropdown-menu dropdown-menu-w px-3"><li><a class="dropdown-item" href="/analysis/transcriptomics/exploratory_analysis/">Exploratory gene expression analysis</a></li><li><a class="dropdown-item" href="/analysis/transcriptomics/rna_seq_differential_expression_analysis/">RNA-seq analysis</a></li><li><a class="dropdown-item" href="/analysis/transcriptomics/pathway_analysis/">Pathway analysis</a></li><li><a class="dropdown-item" href="/analysis/transcriptomics/transcriptome_assembly/">Transcriptome assembly</a></li><li><a class="dropdown-item" href="/analysis/transcriptomics/single_cell_expression_analysis/">Single-cell expression analysis</a></li><li><a class="dropdown-item" href="/analysis/transcriptomics/microRNA_data_analysis/">MicroRNA data analysis</a></li><li><a class="dropdown-item" href="/analysis/transcriptomics/alternative_splicing_analysis/">Alternative splicing analysis</a></li><li><a class="dropdown-item" href="/analysis/transcriptomics/fusion_gene_detection/">Fusion gene detection</a></li></div></details></li></div></details></div></div><script>function mouseOver(x) {document.getElementById("legend").innerHTML = x.title;}</script><div class="color-bg-tertiary-light"><div class="container-lg text-center"><img alt="" src="/assets/img/PCA.png" onmouseover="mouseOver(this)" class="block width-fit" title="Exploratory analysis"></div><div class="container-lg d-flex flex-column justify-content-evenly py-3 px-3"><style>div {text-align: justify; text-justify: inter-word; }</style><h1 class="h2">Principal component analysis (PCA)</h1><p class="f3">Principal component analysis (PCA) is an unsupervised method used to simplify complex, high-dimensional data while still retaining its trends and patterns. This technique is very usefull to summarize genomic data with tens to thousands of dimensions (e.g. genes x samples). Beside PCA, other mathematical approaches exist, t-Distributed Stochastic Neighbor Embedding (t-SNE) and Uniform Manifold Approximation and Projection (UMAP). These two methodologies are better than PCA at preserving elements of the data structure from high-dimensional space. UMAP has been particularly useful to precisely define cell types in mixed populations based on data from single-cell RNA-seq experiments by capturing local relationships within groups of transcriptomes in addition to global relationships between distinct groups.</p><div class="container-lg d-flex flex-column justify-content-evenly pt-4"><h1 class="h2">Available pipelines</h1><div class="container-lg page-section py-2"><div class="d-md-flex flex-wrap gutter flex-auto"><div class="col-lg-6 col-md-6 mb-3 mt-3 d-flex flex-column pt-0"><a href="/analysis/genomics/PCA/" class="d-flex flex-column color-bg-primary rounded-1 box-shadow border text-inherit no-underline flex-1 anim-hover-grow" style="height: 100%"><h3 class="h4 px-3 pt-3">PCA</h3><p class="color-text-secondary lh-condensed px-3 pb-3"></p></a></div><div class="col-lg-6 col-md-6 mb-3 mt-3 d-flex flex-column pt-0"><a href="/analysis/genomics/TSNE/" class="d-flex flex-column color-bg-primary rounded-1 box-shadow border text-inherit no-underline flex-1 anim-hover-grow" style="height: 100%"><h3 class="h4 px-3 pt-3">t-SNE</h3><p class="color-text-secondary lh-condensed px-3 pb-3"></p></a></div><div class="col-lg-6 col-md-6 mb-3 mt-3 d-flex flex-column pt-0"><a href="/analysis/genomics/UMAP/" class="d-flex flex-column color-bg-primary rounded-1 box-shadow border text-inherit no-underline flex-1 anim-hover-grow" style="height: 100%"><h3 class="h4 px-3 pt-3">UMAP</h3><p class="color-text-secondary lh-condensed px-3 pb-3"></p></a></div></div></div></div></div></div></body></html>