---
layout: default
title: Small variant analysis
---

<script> function mouseOver(x) {document.getElementById("legend").innerHTML = x.title;}</script>

<div class="color-bg-tertiary-light">
  <div class="container-lg text-center" >
    <img
      alt=""
      src="/assets/img/rnaseq/principal_component_analysis.small.png"
      onmouseover="mouseOver(this)"
      class="block"
      title="Principal component analysis"
      style="max-width: 300px;"
      />
  </div>

  <div class="container-lg d-flex flex-column justify-content-evenly py-3 px-3">
    <style> div {text-align: justify; text-justify: inter-word; } </style> 
    
    <h1 class="h2">Principal component analysis (PCA)</h1>
    <p class="f3">
      Principal component analysis (PCA) is an unsupervised method used to simplify complex, high-dimensional data while still retaining its trends and patterns. This technique is very usefull to summarize genomic data with tens to thousands of dimensions (e.g. genes x samples). Beside PCA, other mathematical approaches exist, t-Distributed Stochastic Neighbor Embedding (t-SNE) and Uniform Manifold Approximation and Projection (UMAP). These two methodologies are better than PCA at preserving elements of the data structure from high-dimensional space. UMAP has been particularly useful to precisely define cell types in mixed populations based on data from single-cell RNA-seq experiments by capturing local relationships within groups of transcriptomes in addition to global relationships between distinct groups.      
    </p>
  
    <div class="container-lg d-flex flex-column justify-content-evenly pt-4">
      <h1 class="h2">Available pipelines</h1>
      <div class="container-lg page-section py-2">
	<div class="d-md-flex flex-wrap gutter flex-auto">
	  {% assign apps = collections.transcriptomics %}
	  {% for app in apps %}{% if 'rnaseq_principal_component' == app.data.label %}
	  {% include "genomic" app: app.data %}{% endif %}
	  {% endfor %}
	</div>
      </div>
    </div>
  </div>
</div>
