---
layout: default
title: Structural variant analysis
---

<script> function mouseOver(x) {document.getElementById("legend").innerHTML = x.title;}</script>

<div class="color-bg-tertiary-light">
  <div class="container-lg text-center" >
    <img
      alt=""
      src="/assets/img/dnaseq/structural_variants.large.png"
      onmouseover="mouseOver(this)"
      class="block width-fit"
      title="Structural variant analysis"
      />
  </div>

  <div class="container-lg d-flex flex-column justify-content-evenly py-3 px-3">
    <style> div {text-align: justify; text-justify: inter-word;} </style> 
    
    <h1 class="h2">Structural variant analysis</h1>
    <p class="f3">
      Structural variations (SVs) are an extremly diverse category of genetic variations. SVs comprise a myriad of subclasses, from CNVs to rearrangements (inversions and interchromosomal and intrachromosomal translocations). Additionaly, they include mobile element insertions, sergmental duplications and complex rearrangements containing multiple combinations of the above events. Although short-read sequency can resolve SNVs, this sequencing strategy has difficulties in resolving all above SVs structures given the limited sequence and insert size.
    </p>

    <h1 class="h2 pt-4">Workflow</h1>
    <div class="container-lg page-section pt-2">
      <div class="container-lg d-flex flex-column flex-sm-row flex-sm-justify-between flex-sm-items-center py-3 px-3">
	<div class="d-flex flex-self-center" style="flex-basis:10%">
	  <h3 class="h3 text-center mb-auto">Step 1</h3>
	</div>
	
	<div class="d-flex flex-column" style="flex-basis:90%" >
	  <h2 class="h3">Structural variant discovery</h2>
	  <p class="f3">
	    SV detection algorithms leverage signatures that result from mapping discordance between a sample read and the reference genome. Although different heuristics exist, and ensemble approach comprised by containing two or more heuristics has been shown to increase concordance of SV calls when compared with reference data sets.
	  </p>  
	</div>
      </div>

      <div class="container-lg d-flex flex-column flex-sm-row flex-sm-justify-between flex-sm-items-center py-3 px-3">
	<div class="d-flex flex-self-center" style="flex-basis:10%">
	  <h3 class="h3 text-center mb-auto">Step 2</h3>
	</div>
	
	<div class="d-flex flex-column" style="flex-basis:90%" >
	  <h2 class="h3">Filtration of variants</h2>
	  <p class="f3">
	     Following detection, the resultant calls are merged, combining SVs calls uniquely made by each algorithm.
	  </p>  
	</div>
      </div>
      
      <div class="container-lg d-flex flex-column flex-sm-row flex-sm-justify-between flex-sm-items-center py-3 px-3">
	<div class="d-flex flex-self-center" style="flex-basis:10%">
	  <h3 class="h3 text-center mb-auto">Step 3</h3>
	</div>
	
	<div class="d-flex flex-column" style="flex-basis:90%" >
	  <h2 class="h3">Variant annotation</h2>
	  <p class="f3">
	     At this stage, filtered SVs, will be annotated regarding their gene(s)/transcript(s).
	  </p>  
	</div>
      </div>
    </div>
    
    <div class="container-lg d-flex flex-column justify-content-evenly px-3 pt-4">
      <h1 class="h2">Available pipelines</h1>
      <div class="container-lg page-section py-2">
	<div class="d-md-flex flex-wrap gutter flex-auto">
	  {% assign apps = collections.genomics %}
	  {% for app in apps %}{% if 'structural_variants' == app.data.label %}
	  {% include "genomic" app: app.data %}{% endif %}
	  {% endfor %}
	</div>
      </div>
    </div>
  </div>
</div>
