<!doctype html><html lang="en" class="height-full" data-color-mode="auto" data-light-theme="light" data-dark-theme="dark"><head><meta charset="utf-8"><meta name="viewport" content="width=device-width,initial-scale=1"><link rel="stylesheet" href="/assets/dist/styles.css"><link rel="apple-touch-icon" sizes="180x180" href="/apple-touch-icon.png?v=jw3d9vxKo4"><link rel="icon" type="image/png" sizes="32x32" href="/favicon-32x32.png"><link rel="icon" type="image/png" sizes="16x16" href="/favicon-16x16.png"><link rel="manifest" href="/site.webmanifest?v=jw3d9vxKo4"><link rel="mask-icon" href="/safari-pinned-tab.svg?v=jw3d9vxKo4" color="#5bbad5"><link rel="shortcut icon" href="/i3s.ico"><meta name="theme-color" content="#66cce0"></head><body><div class=""><div class="container-lg d-flex flex-items-center flex-justify-between px-3"><div class="py-3 d-inline-flex flex-items-center"><div class="col-1"><a href="/"><img src="/assets/img/i3s.jpg" class="width-fit" alt="i3S logo"></a></div><div class="text-inherit text-left"><h1 class="h4 text-uppercase">Bioinformatic Workflows</h1></div></div><details class="dropdown details-reset details-overlay d-inline-block focus-within"><summary class="h1" aria-haspopup="true">â‰¡</summary><div class="dropdown-menu dropdown-menu-w"><li><details class="dropdown details-reset details-overlay"><summary class="" aria-haspopup="true"><p class="dropdown-item" style="margin-bottom:0">Genomics</p></summary><div class="dropdown-menu dropdown-menu-w px-3"><li><a class="dropdown-item" href="/analysis/genomics/de_novo_sequencing2/">Genome assembly</a></li><li><a class="dropdown-item" href="/analysis/genomics/variant_detection/">Variant analysis</a></li><li><a class="dropdown-item" href="/analysis/genomics/metagenomic_analysis/">Metagenomics</a></li><li><a class="dropdown-item" href="/analysis/genomics/genetic_analysis/">Genetic analysis</a></li></div></details></li><li><details class="dropdown details-reset details-overlay"><summary class="" aria-haspopup="true"><p class="dropdown-item" style="margin-bottom:0">Transcriptomics</p></summary><div class="dropdown-menu dropdown-menu-w px-3"><li><a class="dropdown-item" href="/analysis/transcriptomics/exploratory_analysis/">Exploratory gene expression analysis</a></li><li><a class="dropdown-item" href="/analysis/transcriptomics/rna_seq_differential_expression_analysis/">RNA-seq analysis</a></li><li><a class="dropdown-item" href="/analysis/transcriptomics/pathway_analysis/">Pathway analysis</a></li><li><a class="dropdown-item" href="/analysis/transcriptomics/transcriptome_assembly/">Transcriptome assembly</a></li><li><a class="dropdown-item" href="/analysis/transcriptomics/single_cell_expression_analysis/">Single-cell expression analysis</a></li><li><a class="dropdown-item" href="/analysis/transcriptomics/microRNA_data_analysis/">MicroRNA data analysis</a></li><li><a class="dropdown-item" href="/analysis/transcriptomics/alternative_splicing_analysis/">Alternative splicing analysis</a></li><li><a class="dropdown-item" href="/analysis/transcriptomics/fusion_gene_detection/">Fusion gene detection</a></li></div></details></li></div></details></div></div><script>function mouseOver(x) {document.getElementById("legend").innerHTML = x.title;}</script><div class="color-bg-tertiary-light"><div class="container-lg text-center"><img alt="" src="/assets/img/metagenomesequencing.png" onmouseover="mouseOver(this)" class="block" title="Exploratory analysis" style="max-width: 200px;"></div><div class="container-lg d-flex flex-column justify-content-evenly py-3 px-3"><style>div {text-align: justify; text-justify: inter-word;}</style><h1 class="h2">Whole-metagenome</h1><p class="f3">Whole-metagenome can reach higher resolutions of taxonomical annotation and provide functional gene profiles directly, comparatively with marker gene sequencing. Additionaly, this strategy alleviates biases from primer choice and enables the detection of organisms across all domains of life. The disadvantage of this approach is the cost. Additionally, collections involving a host species should be carefuly considered. Host's DNA can easily be present in higher proportions comparatively with the microbial DNA, which will translate in a lower number of microbial reads which can impair results. Metagenomic classification can be performed by matching reads against a database or assembling reads into contigs, followed by contig match to a database to identify the taxon of each sequence.</p><h1 class="h2 pt-4">Workflow</h1><div class="container-lg page-section pt-2"><div class="container-lg d-flex flex-column flex-sm-row flex-sm-justify-between flex-sm-items-center py-3 px-3"><div class="d-flex flex-self-center" style="flex-basis:10%"><h3 class="h3 text-center mb-auto">Step 1</h3></div><div class="d-flex flex-column" style="flex-basis:90%"><h2 class="h3">Quality control & host removal</h2><p class="f3">In a whole-metagenome analysis the first step is the quality control of the reads by trimming and removing of low quality bases/reads. After this quality control step, in metagenomic experimental steps involving a host species, host reads need to be removed prior to the following analysis. For enviromental samples, this step does not need to be performed.</p></div></div><div class="container-lg d-flex flex-column flex-sm-row flex-sm-justify-between flex-sm-items-center py-3 px-3"><div class="d-flex flex-self-center" style="flex-basis:10%"><h3 class="h3 text-center mb-auto">Step 2</h3></div><div class="d-flex flex-column" style="flex-basis:90%"><h2 class="h3">Read or assemble-based alignment</h2><p class="f3">After quality control, the reads can either be assembled into longer contiguous sequences (assembled into contigs) or passed directly to the taxonomic classifier. The latter is useful for quantitative community profiling and identification of organisms with close relatives on curated databases. When no close relative exist, as can occur in environmental samples, assembly of reads into contigs can be useful since they allow a better assessement of single-copy and conserved genes, taxonomic classification, genome completeness.</p></div></div><div class="container-lg d-flex flex-column flex-sm-row flex-sm-justify-between flex-sm-items-center py-3 px-3"><div class="d-flex flex-self-center" style="flex-basis:10%"><h3 class="h3 text-center mb-auto">Step 3</h3></div><div class="d-flex flex-column" style="flex-basis:90%"><h2 class="h3">Quantification</h2><p class="f3">A feature table is obtained quatifying the frequency of the feature sequences in each sample. To this feature table, a taxonomical assigment will be performed (kingdom, phylum, class, order, family, genus and species).</p></div></div><div class="container-lg d-flex flex-column flex-sm-row flex-sm-justify-between flex-sm-items-center py-3 px-3"><div class="d-flex flex-self-center" style="flex-basis:10%"><h3 class="h3 text-center mb-auto">Step 4</h3></div><div class="d-flex flex-column" style="flex-basis:90%"><h2 class="h3">Diversity analysis</h2><p class="f3">At this stage, alpha and beta diversity will be evalutated to assess microbiota diversity. A phylogenetic tree will be generated based on the detect species. Co-occurence, correlation and discrimination of phylotypes by condition or group will also be carried, if applicable.</p></div></div><div class="container-lg d-flex flex-column flex-sm-row flex-sm-justify-between flex-sm-items-center py-3 px-3"><div class="d-flex flex-self-center" style="flex-basis:10%"><h3 class="h3 text-center mb-auto">Step 5</h3></div><div class="d-flex flex-column" style="flex-basis:90%"><h2 class="h3">Functional annotation</h2><p class="f3">Functional annotation is performed using the genes present in the dataset which are combined into modules and pathways for more comprehensive information.</p></div></div></div><div class="container-lg d-flex flex-column justify-content-evenly px-3 pt-4"><h1 class="h2">Available pipelines</h1><div class="container-lg page-section py-2"><div class="d-md-flex flex-wrap gutter flex-auto"></div></div></div><div class="container-lg d-flex flex-column justify-content-evenly px-3"><h1 class="h2">Follow-up analysis</h1><div class="container-lg page-section pt-2"><div class="container-lg d-flex flex-column flex-sm-row flex-sm-justify-between flex-sm-items-center py-3 px-3"><div class="d-flex flex-self-center" style="flex-basis:30%"><h3 class="h3 mb-auto">Visual representations</h3></div></div></div></div></div></div></body></html>