<h2>RNAseq STAR (Default settings)</h2><p>This pipeline performs the alignment of fastq files, from RNA-seq reads, using the STAR aligner. This software is currently the one with the higher accuracies, even though it requires a large amount of RAM (50Gb for the human genome)</p><h3>What you should provide</h3><ol><li>Reads in fastq format (ideally compressed in .gz format)</li><li>The organism from which they come from.</li></ol><h3>What you will receive</h3><ol><li>File with the aligned reads in BAM format.</li><li>File with gene quantification for the sample.</li><li>File with alignment statistics.</li><li>Quality control plots representing the various factors affecting sample overall quality</li></ol><p><strong>WARNING:</strong> These files are required for followup analysis, such as <a href="../visual_representations/principal_component_analysis/">Principal component analysis</a>, <a href="../visual_representations/heatmap/">Heatmap analysis</a>, <a href="../exploratory_analysis/differentia_expression_analysis/">Differential expression analysis</a> and <a href="../pathway_analysis/">Pathway analysis</a> Visual exploration of the BAM genomic data can be assessed in <a href="https://software.broadinstitute.org/software/igv/">IGV</a>.</p><h3>Software versions</h3><ul><li>STAR v2.7.10a</li><li>BioBamBam2 v2.0.180</li><li>RSeQC v4.0.0</li><li>dupRadar v1.26.1</li></ul><h3>Relevant software values (assume default if unspecified)</h3><ul><li>alignSJoverhangMin set to <strong>8</strong></li><li>alignSJDBoverhangMin set to <strong>1</strong></li></ul><h3>Pipeline</h3><div class="container-lg px-3 pt-3"><div class="page-section d-flex flex-column gutter-md flex-md-row align-items-end mb-2"><div id="vue" style="height: 500px;"><cwl cwl-url="https://raw.githubusercontent.com/i3S-BioinformaticsService/i3s-cwl-ngs-workflows/main/RNA-Seq/rnaseq.star.default.json"></cwl></div><script src="/assets/js/vue.min.js"></script><script src="/assets/js/cwl_svg.js"></script><script>new Vue({
          el: '#vue',
          components: {
              cwl: vueCwl.default
          }
      });</script></div></div><h3>Methods (Please adapt)</h3><p>Mapping to reference genome (assembly version XXX) was performed using BWA mem (version 0.7.17) and duplicates were removed using biobambam2 (version 2.0.87) followed by base quality recalibration using GATK (version 4.3.6.1).</p>